rust/garbage_collector/tests/garbage_collector_prop_test.rs

```diff
@@ -284,4 +353,156 @@ mod tests {
             })?;
         }
     }
+
+    proptest! {
+        #[test]
+        fn test_k8s_integration_gc_version_properties(
```

Comment:
This should be reframed to not be a integration test. This logically can be tested entirely locally.